azkaban-webserver/src/test/java/azkaban/webapp/servlet/MockHttpServletRequest.java

```diff
@@ -0,0 +1,359 @@
+/*
```

Comment:
I did want to use Mockito. I was looking around in the project and didn't see mockito being used. I saw that 'azkaban.project' in azkaban-common is using Mock classes, hence I went with that approach. I will change it to Mockito.